tests/ordering/tests.py

```diff
@@ -90,6 +96,53 @@ def test_order_by_override(self):
             attrgetter("headline")
         )
 
+    def test_order_by_nulls_first_and_last(self):
+        with self.assertRaisesMessage(ValueError,
+                                      "nulls_first and nulls_last are mutually exclusive"):
+            Article.objects.order_by(F("author").desc(nulls_last=True, nulls_first=True))
+
+    def test_order_by_nulls_last(self):
+        Article.objects.filter(headline="Article 3").update(author=self.author_1)
+        Article.objects.filter(headline="Article 4").update(author=self.author_2)
+
+        # check that asc and desc are chainable with nulls_last
+
+        self.assertSequenceEqual(
+            Article.objects.order_by(F("author").desc(nulls_last=True)), [
+                self.a4,
+                self.a3,
+                self.a1,
+                self.a2
+            ])
+
+        self.assertSequenceEqual(
+            Article.objects.order_by(F("author").asc(nulls_last=True)), [
+                self.a3,
+                self.a4,
+                self.a1,
+                self.a2
+            ]
+        )
+
+    def test_order_by_nulls_first(self):
```

Comment:
Please add authors for `article3` and `article4` like in `test_order_by_nulls_last`:
```diff
--- a/tests/ordering/tests.py
+++ b/tests/ordering/tests.py
@@ -125,6 +125,8 @@ class OrderingTests(TestCase):
         )
 
     def test_order_by_nulls_first(self):
+        Article.objects.filter(headline="Article 3").update(author=self.author_1)
+        Article.objects.filter(headline="Article 4").update(author=self.author_2)
         self.assertSequenceEqual(
             Article.objects.order_by(F("author").asc(nulls_first=True)), [
                 self.a1,
@@ -138,8 +140,8 @@ class OrderingTests(TestCase):
             Article.objects.order_by(F("author").desc(nulls_first=True)), [
                 self.a1,
                 self.a2,
-                self.a3,
-                self.a4
+                self.a4,
+                self.a3
             ]
         )
```